suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -421,6 +421,20 @@ export const switchDuplicatedDevice = createThunk(
     },
 );
 
+// Sort devices by timestamp and put Portfolio Tracker device at the end.
+const sortDevices = (devices: TrezorDevice[]) => {
```

Comment:
This is not a thunk, just plain util function so I would consider moving it to `suite-common/suite-utils/src/device.ts`

Reply:
But it makes sense only in context of this thunk, I would prefer to keep it here until it's reused somewhere.